app/components/QuoteForm.tsx

```diff
@@ -55,6 +101,8 @@ export default function QuoteForm({
             _hover={{
               borderColor: "brand.accentGreen",
             }}
+            name="name"
```

Comment:
Yes this is from react hook forms - it's how it links the field to the object key name I believe. I'm also not seeing any lint etc. here on my end

Reply:
As I mentioned in another comment, this only confirms that our setups differ in some way. I finally read what the type error says, and it simply says that line 104 is unnecessary, because on line 105 you're overwriting it (with the same value, but still). So you can simply remove this line
![Screen Shot 2024-02-27 at 7 35 23 PM](https://github.com/wavecrestweb/kenworthy-machine/assets/109827834/eaf19d25-71cf-4e7e-8f3a-1917e60ecdee)